reference/lexical_analysis.po

```diff
@@ -877,110 +1192,151 @@ msgid ""
 "fields. The :ref:`format specifier mini-language <formatspec>` is the same "
 "as that used by the string .format() method."
 msgstr ""
+"Les spécifications de format peuvent inclure des champs de remplacement "
+"imbriqués. Ces champs imbriqués peuvent inclure leurs propres champs de "
+"conversion et :ref:`spécifications de format <formatspec>` mais "
+"l'imbrication ne doit pas aller plus profond. Le :ref:`mini-langage de "
+"spécification de format  <formatspec>` est le même que celui utilisé par la "
+"méthode string.format()."
```

Comment:
I don't like much "string.format()" here, string is another module (without format). Maybe use `str.format()` if you want to keep it that way, or use "celui utilisé par la méthode `format()` des chaînes"?